schema.sql

```diff
@@ -1,5 +1,16 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals
+(
+    id integer NOT NULL,
+    name character varying(100) NOT NULL,
+    date_of_birth date NOT NULL,
+    escape_attempts integer DEFAULT 0,
+    neutered boolean NOT NULL,
+    weight_kg numeric
+)
```

Comment:
- [OPTIONAL] I think it would be better if mark the id column as the primary key to be used as a unique identifier like in **[this example](https://www.geeksforgeeks.org/postgresql-primary-key/)**.